### DISCUSSION TITLE
/assets/pdf-4.0.851.min.mjs Request Method: GET Status Code: 404 Not Found

### DISCUSSION THREAD

**Discussion by venkycoder422** [stephanrauh/ngx-extended-pdf-viewer#2225] [Q&A] [answered]
![issue](https://github.com/stephanrauh/ngx-extended-pdf-viewer/assets/76251822/8dd4e7cd-11fc-46f7-a9b0-38b49771d3ba)

The ngx-extended-pdf-viewer working locally after I deploy the code, I am getting BASEURL/assets/pdf-4.0.851.min.mjs Request Method: GET Status Code: 404 Not Found 

please tell me how do I solve this problem

**Comment by stephanrauh** [comment]
No idea... but I'd start with the simple stuff. Maybe the file really isn't there. What happens when you open the link in a new tab?

**Comment by Cheilanthe** [comment]
Hi, i have the same issue, but in my local project. How did you fix it? Did you find a solution?

**Reply by venkycoder422** [reply, marked as answer]
Hi @Cheilanthe in your project make sure to allow the mjs files to load. on web.config file you have to write this `  <mimeMap fileExtension=".mjs" mimeType="text/javascript" />
`

If any doubt replay here

**Comment by kr2792** [comment]
My problem was that this part

`              {
                "glob": "**/*",
                "input": "node_modules/ngx-extended-pdf-viewer/assets/",
                "output": "/assets/"
              },`

Had somehow been removed from assets section in the angular.json file.
